### PR TITLE
PRO-1569 Change app logo and The colour change for transaction direction

### PR DESCRIPTION
**Pull request by lalitkumarrock**
https://linear.app/pillarproject/issue/PRO-1569/pillar-wallet-update-app-icon

## Description

- Updated App Icon (Android + iOS)
- Fixed small issues

## How Has This Been Tested?

- Tested on Simulator 14 PRO
- Tested on Android Device
-

## Screenshots (if appropriate):
Simulator Screenshot - iPhone 14 Pro - 2023-06-26 at 21 46 48

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/screens/Asset/TradingActivity.tsx [hunk 1]
```diff
@@ -60,6 +60,8 @@ const TradingActivityScreen = () => {
     );
   };
 
+  const getKeyExtractor = (item, index) => item.amm + index.toString();
```
Comment (IAmKio) [inline src/screens/Asset/TradingActivity.tsx:63]: Repeated code, this should also be in a utils file from what it looks like

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just 1 small change.

In future, please split these PRs up as there are a few issues here rolled into one (the logo change, the colour change for transaction direction etc) and it's bad practice.

Please update the title to indicate what has been changed.

File: src/screens/Asset/TradingActivity.tsx (hunk 1 above)
Comment (lalitkumarrock) [reply src/screens/Asset/TradingActivity.tsx]: Updated it!

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**